openhands/core/config/app_config.py

```diff
@@ -134,4 +134,5 @@ def get_agent_configs(self) -> dict[str, AgentConfig]:
     def model_post_init(self, __context):
         """Post-initialization hook, called when the instance is created with only default values."""
```

Comment:
```suggestion
        """Post-initialization hook, called when the instance is created."""
```
Unrelated to the PR, it looks like this was inaccurate